src/calculateRentalCost.js

```diff
@@ -3,8 +3,22 @@
  *
  * @return {number}
  */
+
+const LONG_TERM = 7;
+const MID_TERM = 3;
+const LONG_TERM_DISCOUNT = 50;
+const MID_TERM_DISCOUNT = 20;
+
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40 * days;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  } else if (days >= MID_TERM) {
```

Comment:
Here will better use if without else

Reply:
5. DON'T add `else` after `if` with `return` - the code after it won't be executed anyway.